sync/downstream.py

```diff
@@ -141,9 +141,22 @@ def update_status(self, action, merge_sha=None, wpt_base=None):
             self.status = "open"
             self.pr_status = "open"
 
-    def update_wpt_head(self):
+    def update_wpt_commits(self):
         if not self.wpt_commits.head or self.wpt_commits.head.sha1 != self.pr_head.sha1:
             self.wpt_commits.head = self.pr_head
+        if len(self.wpt_commits) == 0 and self.git_wpt.is_ancestor(self.wpt_commits.head.sha1,
+                                                                   "origin/master"):
+            # The commits landed on master so we need to change the commit
+            # range to not use origin/master as a base
+            base_commit = None
+            assert self.wpt_commits.head.pr() == self.pr
```

Comment:
Do we need some docs on what to do if this assertion is hit?

Reply:
I don't really know what they would say. That would indicate a bug because we would be getting the wrong commits for a pr.